### PR TITLE
28 create database tables

### PR DESCRIPTION
**Pull request by BingHawk**
- Added more tables to the database (contact and users).
- Filled the city table with coordinate data from OSM. (95 of 120 cities in Open AQ now have their coordinates in the database). 
- Moved Database code to separate files.
- Added a function in the app to read the city coordinates from the database.
- Reorganized the app to do some requests before the app starts, meaning that querying the /cities endpoint will be super fast and that the data from the database is available as global variables in the different "@app.route..." functions.

### CODE REVIEW

File: flaskVueBackend/osm.py [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+            if filter is None or node.tags['name'].title() in filter:
+                c = {
+                    'name': node.tags['name'].title(),
+                    'coordinates':[float(node.lon), float(node.lat)]
+                }
+                cities.append(c)
+        return cities
+
+
+if __name__ == "__main__":
```
Comment (BingHawk) [inline flaskVueBackend/osm.py:51]: This code is just for testing what is above, it will not be used

File: flaskVueBackend/osm.py [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+        for node in res.nodes:
+            if filter is None or node.tags['name'].title() in filter:
+                c = {
+                    'name': node.tags['name'].title(),
+                    'coordinates':[float(node.lon), float(node.lat)]
+                }
+                cities.append(c)
+        return cities
+
+
```
Comment (BingHawk) [inline flaskVueBackend/osm.py:50]: The following code is just for testing what is above, it will not be used

File: flaskVueBackend/app.py [hunk 3]
```diff
@@ -1,15 +1,85 @@
...
+MYPWRD = '123456'
+MYPORT = '5433'
+
+#### FLASK CONFIGURATION
 app = Flask(__name__)
 app.config["DEBUG"] = True
 app.config["APPLICATION_ROOT"] = "/"
 cors = CORS(app, resources={r"/api/*": {"origins": "*"}})
 
+#### UNROUTED FUNCTIONS
```
Comment (BingHawk) [inline flaskVueBackend/app.py:18]: Get city names is just moved here, not changed

File: flaskVueBackend/app.py [hunk 4]
```diff
@@ -34,34 +104,11 @@ def get_locations():
 #EndPoint for the cities
 @app.route('/api/cities') # Moved the actual API call to another function to be able to reuse get_cityNames() elsewhere. 
 def serve_cityNames():
-    response, cityDict = get_cityNames()
-    return response
+    return cityResponse
```
Comment (BingHawk) [inline flaskVueBackend/app.py:108]: GetCityNames is just moved away from here

**Review by AlbaLunner — COMMENTED** [review]
The initDb.py is understandable, I think everything makes sense. Comparing with prof. di nittos slides it looks correct. If you can run it I think it's finished. 
The osm.py is also understandable, but I didn't try to run either file, can do that soon if you want.

**Pull request merged**